main.cpp

```diff
@@ -52,84 +52,109 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: bulldozer
+//  action 1: the blade goes up
+blade.up();
 
-//  action 2:
+//  action 2: the blade goes down
+blade.down();
 
-//  action 3:
+//  action 3:the bulldozer drives forward
+drive.forward();
 
 //  2)
-//  Noun:
-//  action 1:
+//  Noun:crane
+//  action 1:rotate to the left
+rotate.left();
 
-//  action 2:
+//  action 2:rotate to the right
+rotate.right();
 
-//  action 3:
+//  action 3: raise the pulley and hook slowly
+raise.slow();

```

Comment:
the noun is `crane`, not `rotate` or `raise`
`crane.rotateLeft();`
`crane.rotateRight();`
`crane.raiseSlow();`
- [ ] revise